src/crypto/PSAOperationalKeystore.cpp

```diff
@@ -17,13 +17,22 @@
 
 #include "PSAOperationalKeystore.h"
 
+#include <lib/core/TLV.h>
 #include <lib/support/CHIPMem.h>
+#include <lib/support/DefaultStorageKeyAllocator.h>
 
 #include <psa/crypto.h>
 
 namespace chip {
 namespace Crypto {
 
+namespace {
+// Tags for our operational keypair storage copied from PersistentStorageOperationalKeystore.cpp
```

Comment:
Shouldn't there be comments in PersistentStorageOperationalKeystore about how if something there changes this code needs to be updated?